server.go

```diff
@@ -2314,8 +2314,12 @@ func (s *server) peerHandler() {
 	}
 
 	if !cfg.DisableDNSSeed {
+		requiredFeatures := wire.SFNodeNetwork | wire.SFNodeWitness
+		if cfg.Utreexo {
+			requiredFeatures = wire.SFNodeUtreexo
```

Comment:
This doesn't look right. It'd be dropping `SFNodeNetwork` here.

Reply:
You're right. Fixed in 0772c48